metron-interface/metron-alerts/src/app/alerts/configure-table/configure-table.component.ts

```diff
@@ -91,6 +95,46 @@ export class ConfigureTableComponent implements OnInit {
     });
   }
 
+  ngAfterViewInit() {
+    fromEvent(this.filterColResults.nativeElement, 'keyup')
+      .pipe(debounceTime(250))
+      .subscribe(e => {
+        this.filterColumns(e['target'].value);
+      });
+  }
+
+  filterColumns(val) {
+    const words = val.trim().split(' ');
+    this.filteredColumns = this.allColumns.filter(col => {
+      if (col.displayName) {
+        if (this.colMissingFilterKeyword(words, col, true)) {
+          return false;
+        } else {
+          return true;
+        }
+      } else {
+        if (this.colMissingFilterKeyword(words, col)) {
+          return false;
+        } else {
+          return true;
+        }
+      }
+    });
+  }
```

Comment:
It's way explicit, which is a good thing in general but in my opinion, we can save a few extra lines here:

```
filterColumns(val) {
  const words = val.trim().split(' ');
  this.filteredColumns = this.allColumns.filter(col => {
    return !this.colMissingFilterKeyword(words, col, col.displayName)
  });
}
```